AISKU/rollup.config.js

```diff
@@ -23,7 +23,7 @@ updateDistEsmFiles(replaceValues, banner, true, true, "dist-es5");
 
 export default createConfig(banner, 
   {
-    namespace: "Microsoft.ApplicationInsights3",
+    namespace: "Microsoft.ApplicationInsights",
```

Comment:
This is now the setting for the "global" namespace, previously we had some code during the module initialization that would "reassign" (copy a reference) to the original global namespace for backward compatibility.